app-emacs/mmm-mode/mmm-mode-0.5.1.ebuild

```diff
@@ -16,6 +16,13 @@ KEYWORDS="amd64 ppc x86 ~amd64-linux ~x86-linux ~ppc-macos"
 
 SITEFILE="50${PN}-gentoo.el"
 
+src_prepare() {
+	# fix bug 561306
+	for file in configure elisp-comp; do
+		sed -i -e "s/-batch/-batch --no-site-file/g" $file || die
+	done
```

Comment:
Oh, thanks, I thought can't accept more then one file and didn't even try. I'll know for the future.